src/Donations/Micros/DonationTypes/BouquetDonations.tsx

```diff
@@ -105,6 +116,8 @@ function FundingDonations({ setopenCurrencyModal }: Props): ReactElement {
     }
   }, [paymentSetup]);
 
+  const approximatelyEqual = (v1, v2, epsilon = 0.001) =>
+    Math.abs(v1 - v2) < epsilon;
```

Comment:
This function is also defined in common.ts, It can be removed and imported from there.